core/src/data/value/big_endian.rs

```diff
@@ -46,6 +46,7 @@ impl Value {
                 .chain(v.as_bytes().iter())
                 .copied()
                 .collect::<Vec<_>>(),
+            Value::Bytea(v) => v.to_vec(),
```

Comment:
If you need to clone ownership from `&Vec<_>` to `Vec<_>`,  `clone` looks fine.

Reply:
I also get used to use `clone` but I just saw that compiler recommends to use `to_vec()` in this kind of cases (but it seems that though I use `clone`, clippy accepts, too)
Hmm, but It would be good to use `to_vec()` in here.

`to_vec` - Copies self into a new Vec.
description fits well to here

https://doc.rust-lang.org/stable/std/vec/struct.Vec.html#method.to_vec